ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -180,6 +187,12 @@ impl Blockstore {
                     to_slot,
                 )?;
             }
+            PurgeType::CompactionFilter => {
+                // this purge type completely and indefinitely relies on the proper working of compaction
```

Comment:
Nit: Prefix with a clarifying sentence:  "No explicit action is required here because this purge type..."